packages/common/sagas/entities/item.js

```diff
@@ -408,9 +407,9 @@ export function* createItemSaga({
           generateSystemItem,
           ENTITY_TYPE.ITEM,
           userPersonalDefaultListId,
-          itemId,
+          itemData.id,
         );
-        systemItemData.relatedItem = itemId;
+        systemItemData.relatedItem = itemData.id;
```

Comment:
Should be relatedItemId instead of relatedItem.
Or send an object.

Reply:
Updated to relatedItemId